hooks/ruby/delete-repository-event/app.rb

```diff
@@ -0,0 +1,61 @@
+# Hook example for notifying an administrator in a repository by creating an issue when a repository is deleted.
+#
+# Needs the following environment variables
+#   GITHUB_HOST - the domain of the GitHub Enterprise instance. e.g. github.example.com
+#   GITHUB_API_TOKEN - a Personal Access Token that has the ability to create an issue in the notification repository.
+#   GITHUB_NOTIFICATION_REPOSITORY - the repository in which to create the nofication issue. e.g.
+#
+# Dependencies:
+#   octokit   - https://github.com/octokit/octokit.rb
+#   sinatrarb - http://www.sinatrarb.com/
+
+require 'octokit'
+require 'sinatra'
+require 'json'
+
+enable :logging
+github_api_token               = ENV['GITHUB_API_TOKEN']
+github_notification_repository = ENV['GITHUB_NOTIFICATION_REPOSITORY']
+github_host_url                = ENV['GITHUB_HOST']
+github_api_endpoint            = "https://#{github_host_url}/api/v3"
+
+Octokit.configure do |c|
+  c.api_endpoint = github_api_endpoint
+  c.access_token = github_api_token
+end
+
+# Needed so that the webhook setup passes
+post '/' do
+  200
+end
+
+# When receiving a webhook for repository deletion (https://developer.github.com/v3/activity/events/types/#repositoryevent)
+#   create an issue in the `github_notification_repository` set by environment variable
+post '/delete-repository-event' do
+  logger.info github_api_token
```

Comment:
I'm hesitant to print an API tokens to console/logs. I'd hope that anyone running this in a production environment would take a look at the `logging` settings.